app/templates/components/footer-bar.hbs

```diff
@@ -0,0 +1,8 @@
+<div class="container">
+  {{#link-to "home"  (query-params author=null tag=null page=1) class="logo-font" }}
```

Comment:
Hope this isn't too annoying but there are two spaces between `"home"` and `(query-params...` 😅